nmproxy/nm-proxy.go

```diff
@@ -38,3 +40,8 @@ func Start(ctx context.Context, mgmChan chan *nm_models.HostPeerUpdate, stunAddr
 	go manager.Start(ctx, mgmChan)
 	server.NmProxyServer.Listen(ctx)
 }
+
+// IsProxyRunning - fetches the proxy status
```

Comment:
why do we need this func; can't one just call config.GetCfg().IsProxyRunning  ?

Reply:
if the func is only called once, I agree, no need